CRM/Custom/Import/Form/MapField.php

```diff
@@ -24,6 +23,9 @@ public function preProcess() {
     $this->_columnCount = $this->get('columnCount');
     $this->assign('columnCount', $this->_columnCount);
     $this->_dataValues = $this->get('dataValues');
+    $highlightedFields = array();
+    $highlightedFields[] = 'contact_id';
+    $highlightedFields[] = 'external_identifier';
```

Comment:
I agree, but again: I just the same form from the contact import code. Do you want me to change it here already?